### PR TITLE
Telemetry and metrics

### PR DESCRIPTION
**Pull request by wiggin77**
#### Summary
This PR adds additional telemetry (rudder) and metrics (prometheus).  Also fixes an issue where existing user count telemetry was only updated on startup and never refreshed.

- total blocks by block type
- total workspaces 
- blocks activity (insert/delete)
- login success / fail

Metrics also periodically logged using "metrics" level.

#### Ticket Link
https://focalboard-community.octo.mattermost.com/workspace/qgsck6cts3fwpqwyjiupjm5cde?id=47aa9bb4-6967-4a96-83c7-11bd6b20f1eb&v=ca1a5441-4c1c-4d0e-856e-88cc744576c8&c=a8088f01-d964-4369-8330-2e4b2a1aab5f

### CODE REVIEW

**Review by jespino — APPROVED** [review]
LGTM

File: server/app/app.go [hunk 1]
```diff
@@ -11,32 +12,39 @@ import (
 	"github.com/mattermost/mattermost-server/v5/shared/filestore"
 )
 
+type AppServices struct {
```
Comment (jespino) [inline server/app/app.go:15]: good point :)

File: server/server/server.go [hunk 2]
```diff
@@ -120,82 +142,41 @@ func New(cfg *config.Configuration, singleUserToken string, logger *mlog.Logger)
...
-			"serverRoot":  cfg.ServerRoot == config.DefaultServerRoot,
-			"port":        cfg.Port == config.DefaultPort,
-			"useSSL":      cfg.UseSSL,
-			"dbType":      cfg.DBType,
-			"single_user": len(singleUserToken) > 0,
-		}
-	})
-	telemetryService.RegisterTracker("activity", func() map[string]interface{} {
-		return map[string]interface{}{
-			"registered_users":     registeredUserCount,
```
Comment (jespino) [inline server/server/server.go:179]: :facepalm

**Review by chenilim — APPROVED** [review]
LGTM! 👍

**Pull request merged**